packages/forms/frontend/sirius-components-forms/src/views/DetailsView.tsx

```diff
@@ -10,11 +10,17 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
-import { WorkbenchViewComponentProps } from '@eclipse-sirius/sirius-components-core';
-import { useDetailsViewConfiguration } from './DetailsViewConfiguration';
+import { WorkbenchViewComponentProps, useData } from '@eclipse-sirius/sirius-components-core';
+import { detailsViewFormConverterProviderExtensionPoint } from './DetailsViewExtensionPoints';
 import { FormBasedView } from './FormBasedView';
 
 export const DetailsView = (props: WorkbenchViewComponentProps) => {
-  const { converter } = useDetailsViewConfiguration();
-  return <FormBasedView {...props} subscriptionName="propertiesEvent" converter={converter} />;
+  const { data: converterProvider } = useData(detailsViewFormConverterProviderExtensionPoint);
+  return (
+    <FormBasedView
+      {...props}
+      subscriptionName="propertiesEvent"
+      converter={converterProvider.converter(props.editingContextId)}
```

Comment:
The API could be simpler by providing the editingContextId while converting

Reply:
There's no need to separate the conversion in two parts (providing a converter vs doing the work) just do nothing if you're not involved.

Reply:
We can also provide this API for any for based view not just the details one, I'll update the API to make it easier to do so

Reply:
I'll also update things on the tree side of this contribution to make things easier to manipulate

Reply:
In both case, the new API now supports multiple form and tree converters